public/app.js

```diff
@@ -0,0 +1,518 @@
+                          </div>
+                          <div class="col-9 column-right">
+                            <h5>${applicant.playerName.firstName} ${applicant.playerName.lastName}</h5>
+                            <h6>Classes: <small>${classesToDisplay}</small></h6>
+                            <p>Tilde drinking vinegar pok pok, swag raw denim pork belly crucifix raclette air plant authentic kinfolk wolf helvetica synth thundercats. Neutra copper mug 8-bit gochujang. Af venmo vexillologist fashion axe, narwhal schlitz artisan portland.</p>
+                            <ul class="team-member-controls nav-list">
+                              <li><a class="button-good js-team-accept" href="#">Accept</a></li>
+                              <li><a class="button-bad js-team-reject" href="#">Reject</a></li>
+                            </ul>
+                          </div>
+                          <div class="col-12">
+                            <form class="role-select-form" hidden>
+                              <fieldset>
+                                <select name="select-class" required="true">
+                                  ${stringOfClassOptions}
+                                </select>
+                                <input id="tank-btn-${applicant._id}" type="radio" name="team-role" value="Tank" required />
+                                <label for="tank-btn">Tank</label> |
+                                <input id="healer-btn-${applicant._id}" type="radio" name="team-role" value="Healer" />
+                                <label for="healer-btn">Healer</label> |
+                                <input id="dps-btn-${applicant._id}" type="radio" name="team-role" value="DPS" />
+                                <label for="dps-btn">DPS</label> |
+                                <input type="submit" value="Submit">
+                                <button class="cancel-btn"type="button" name="cancel-btn">Cancel</button>
+                              </fieldset>
+                            </form>
+                          </div>
+                        </div>
+                      </div>`;
+    });
+    $('.team-members-content').html(applicantsPosts);
+  }
+
+
+  if (state.activePage === 'home') {
+    renderHeader();
+    renderHomePage();
+  } else if (state.activePage === 'account-my-profile'){
+    renderHeader();
+    renderMyProfile();
+  } else if (state.activePage === 'account-myteam-members') {
+    renderHeader();
+    renderAccountPage();
+    addMembersToAccountPage();
+  } else if (state.activePage === 'account-myteam-applicants') {
+    renderHeader();
+    renderAccountPage();
+    addApplicantsToAccountPage();
+  } else if (state.activePage === 'signup') {
+    renderHeader();
+    renderSignupPage();
+  }
+}
+
+// IN WHICH WE HANDLE THE EVENTS
+
+// nav bar links
+function eventHandlers() {
```

Comment:
I think you can avoid having to constantly re-call this function by using event delegation, i.e.

```
$('body').on('click', ''#account-link'', e => {
```

That kind of event will persist across re-renders.